### DISCUSSION TITLE
Single-project MSIX packaging tool is rejected by partner center

### DISCUSSION THREAD

**Discussion by sowenzhang** [microsoft/WindowsAppSDK#1925] [Q&A]
I am getting this from partner center 

`package acceptance validation error: The package {package}_1.0.0.0_x64.msix uses an unsupported version of file makepri.exe (10.0.22523.1000). The following versions are allowed: < 10.0.10500.0; >= 10.0.10586.0 & < 10.0.11000.0; >= 10.0.14383.0 & < 10.0.14800.0; >= 10.0.15053.0 & < 10.0.15100.0; >= 10.0.16299.0 & < 10.0.16350.0; >= 10.0.17134.0 & < 10.0.17500.0; >= 10.0.17763.0 & < 10.0.18199.0; >= 10.0.18362.0 & < 10.0.18799.0; >= 10.0.19041.0 & < 10.0.19399.0; >= 10.0.20348.0 & < 10.0.21242.0; >= 10.0.22000.194 & < 10.0.22350.0. Please update your Visual Studio build tools and try again.` 

I tried this extension in 2019 and 2022. 

I feel I will be told this is a partner center's issue, but I didn't have this problem with WindowsAppSDK 0.8, when having a separate package project. This new extension is supposed to help us make it easier to create an app package to submit in Store, without dealing with partner center support, right? 

Unfortunately, this version of `makepri` comes from the extension 

![image](https://user-images.githubusercontent.com/2231315/146985172-484a3698-2f0a-45ef-9c52-202b5afddf35.png)

Can we have a work around?

**Comment by sowenzhang** [comment]
BTW, my workaround is updating this line in the project file to NOT using the preview version, instead, pick another one in my machine

`<PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22000.196" />` 

This means, the extension is not good for publishing apps to Store. Not sure if it's me missing from the document.

**Comment by Zulu-Inuoe** [comment]
In case somebody else lands here, there is a discussion of this issue at https://github.com/microsoft/WindowsAppSDK/issues/4480
Currently the solution is in fact to downgrade the build tools
